assets/scss/base/_content.scss

```diff
@@ -337,6 +337,7 @@ samp {
   font-family: $font-monospace;
   background-color: #fff;
   margin: 0 auto;
+  margin-top: 14px;
```

Comment:
Pour moi, on devrait le mettre seulement pour deux blocs qui se suivent (cf. l'issue).

Reply:
Je ne suis pas sur de comprendre. Qu'est ce que tu veux dire ?

Reply:
La subtilité de l'issue était dans le signe '+': https://www.it-swarm.dev/fr/css/que-signifie-le-selecteur-css-signe-plus/967297211/